recipes/monet/meta.yaml

```diff
@@ -1,60 +0,0 @@
-{% set name = "monet" %}
-{% set version = "2.1.5" %}
```

Comment:
could you please revert it?
Please do not delete the folders on ``recipes``
Because we may face some problems

Reply:
This was to another recipe that I had.  It is not part of the standard conda-forge/staged-recipes